CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ...
 
+## 1.8.6 (13.04.2021)
+
+#### ([1336](https://github.com/allegro/hermes/pull/1336)) Added headers provider for batch sender
```

Comment:
To be consistent with the rest of the changelog I suggest to add `Enhancements` section.